.azure-pipelines/build-template.yml

```diff
@@ -84,38 +116,39 @@ jobs:
       artifact: ${{ parameters.swss_common_artifact_name }}
       runVersion: 'latestFromBranch'
       runBranch: 'refs/heads/$(BUILD_BRANCH)'
```

Comment:
Looks like the swsscommon dash branch doesn't differ from the master branch - will this change in the future? Should we continue to use master branch here since the swsscommon job keeps failing due to the sairedis repo not having a dash branch?

Reply:
My understanding is that swsscommon for dash will always be on master branch. @prsunny to confirm.

Reply:
Discussed with @prsunny and @prabhataravind, we should always be using the master branch of swsscommon - can you add a new parameter like `swss_common_branch_name` so we can specify this for DASh jobs?

Reply:
Done